articles/aks/use-wasi-node-pools.md

```diff
@@ -58,15 +58,15 @@ az provider register --namespace Microsoft.ContainerService
 
 ## Limitations
 
-* Currently, there are only containerd shims available for [spin][spin] and [slight][slight] applications, which use the [wasmtime][wasmtime] runtime. In addition to wasmtime runtime applications, you can also run containers on WASM/WASI node pools.
-* You can run containers and wasm modules on the same node, but you can't run containers and wasm modules on the same pod.
```

Comment:
you can actually run Linux and Wasm containers side-by-side with the latest version now!